students/ZackConnaughton/session04/file_exercise.py

```diff
@@ -0,0 +1,46 @@
+#!/user/bin/env python
+import os
+
+def copy_file(file):
+    outfile = open('destination/' + file, 'wb')
+    x = 1
+    for line in open('source/' + file, 'rb'):
+        print(x)
+        x += 1
+        outfile.write(line)
+    outfile.close()
+
+def file_ex():
+    print(os.listdir())
+    copy_file('document.txt')
+    copy_file('pic.jpg')
+    copy_file('mp.jpg')
+
+
+def reset_destination():
+    files = ('document.txt', 'pic.jpg', 'mp.jpg')
+    for f in files:
+        if os.path.exists('destination/' + f):
+            os.remove('destination/' + f)
+
+
+def file_parse():
+    students = open('students.txt')
+    program_dict = {}
+    program_set = set()
```

Comment:
why both the dict and the set? the keys in the dict are the same as the set. yes?